widgetry/src/color.rs

```diff
@@ -47,11 +47,12 @@ pub enum Fill {
 }
 
 #[derive(Debug, Copy, Clone, PartialEq)]
-pub struct Texture(u32);
+pub struct Texture(pub u32);
 
 #[allow(dead_code)]
 impl Texture {
     pub const NOOP: Texture = Texture(0);
+    // Note all of these are based on the default built-in spritesheet
```

Comment:
Future extensions could manage multiple spritesheets, make it easier to ID individual sprites, etc. But none of that is needed for now